universal-login-sdk/test/core/observers/BalanceObserver.test.ts

```diff
@@ -141,7 +145,7 @@ describe('INT: BalanceObserver', () => {
     });
 
     afterEach(async () => {
-      await balanceObserver.stop();
+      // await balanceObserver.stop();
```

Comment:
why?

Reply:
oh thanks, because we stopping observer when we unsubscribe all subscriptions